### PR TITLE
Adding VisionDemo project

### PR DESCRIPTION
**Pull request by pburma**
VisionDemo, collection of small examples for TensorFlow and VisionScripts.

### CODE REVIEW

**Review by JakeSchmitz — CHANGES_REQUESTED** [review]
Will this work with the version running on dev currently, or does this require 1.27?

**Comment by pburma** [comment]
It was built on 1.26.

File: VisionDemo/procedures/vsExample.vail [hunk 1]
```diff
@@ -0,0 +1,33 @@
+PROCEDURE vsExample(imageName String)
+
+    // Create the script
```
Comment (JakeSchmitz) [inline VisionDemo/procedures/vsExample.vail]: For consistency with your other procedures, can you un-indent this?

**Review by JakeSchmitz — DISMISSED** [review]
LGTM, just one style comment to fix before merging

**Review by JakeSchmitz — APPROVED** [review]
Hey @pburma really sorry this never got merged in. I've approved it now. If you want to test it out again on 1.28 feel free, or merge this in directly.

**Comment by JakeSchmitz** [comment]
Closing because this was taken over by our intern @jvic12 and finally merged into master (in time for the release of 1.31)

**Pull request closed without merging**